From b888411b042df9414d1d78d99332b672e65c4eb9 Mon Sep 17 00:00:00 2001
From: Bob Copeland <copeland@lastpass.com>
Date: Wed, 16 May 2018 16:07:58 -0400
Subject: [PATCH] Revert "pins: remove GlobalSign R1/R3 pins"

This reverts commit 46e2a0fb488f4ed7fd8cc3f1c5b64ca009abc5ba.
---
 pins.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pins.h b/pins.h
index e629b6f..7455574 100644
--- a/pins.h
+++ b/pins.h
@@ -5,8 +5,12 @@ const char *PK_PINS[] = {
 	"HXXQgxueCIU5TTLHob/bPbwcKOKw6DkfsTWYHbxbqTY=",
 	/* current lastpass.eu primary (AddTrust) */
 	"lCppFqbkrlJ3EcVFAkeip0+44VaoJUymbnOaEUk7tEU=",
+	/* future lastpass root CA (GlobalSign R1) */
+	"K87oWBWM9UZfyddvDfoxL+8lpNyoUB2ptGtn0fv6G2Q=",
 	/* future lastpass root CA (GlobalSign R2) */
 	"iie1VXtL7HzAMF+/PVPR9xzT80kQxdZeJ+zduCB3uj0=",
+	/* future lastpass root CA (GlobalSign R3) */
+	"cGuxAXyFXFkWm61cF4HPWX8S0srS9j0aSqN0k4AP+4A=",
 	/* future lastpass.com primary (leaf) */
 	"0hkr5YW/WE6Nq5hNTcApxpuaiwlwy5HUFiOt3Qd9VBc=",
 	/* future lastpass.com backup (leaf) */
